src/sentry/static/sentry/app/views/app.jsx

```diff
@@ -177,7 +174,14 @@ class App extends React.Component {
 
   updateTracing() {
     const route = getRouteStringFromRoutes(this.props.routes);
-    Tracing.startTrace(getCurrentHub(), route);
+    const scope = getCurrentHub().getScope();
+    if (scope) {
+      const transactionSpan = scope.getSpan();
```

Comment:
Could this potentially get a span inside the transaction span?

Reply:
Theoretically yes but, `scope.getSpan` always returns the parent span which always should be the transaction.